app/backplane/radio_module/include/n_radio_module_types.h

```diff
@@ -0,0 +1,22 @@
+#ifndef N_RADIO_MODULE_TYPES_H
+#define N_RADIO_MODULE_TYPES_H
+
+#include <stdint.h>
+#include <f_core/utils/n_gnss_utils.h>
+
+namespace NRadioModuleTypes {
+
+struct __attribute__((packed)) RadioBroadcastData {
+    uint16_t port;
+    uint8_t data[256 - sizeof(uint16_t)];
```

Comment:
whats the significance of  256-sizeof(u16)? is that something that lora mandates or just the format we agreed on

Reply:
Format we've agreed on forever now